src/app/shared/components/Input/index.tsx

```diff
@@ -95,6 +97,7 @@ export default function Input({
           setIsFocused(false);
         }}
         value={value}
+        name={name}
```

Comment:
i'm just curious if `name` is undefined will it render name="undefined" in the html 🤔  or it won't render name at all. 

If it is the former then there is no problem with this approach.

Reply:
does not render name